docs-src/typescript/chapter-project-setup/backgroundcheck-boilerplate-start-workflow.md

```diff
@@ -0,0 +1,186 @@
+---
+id: backgroundcheck-boilerplate-start-workflow
+title: Start Workflow using the CLI
+sidebar_label: Start Workflow
+description: Learn how to start a Temporal Workflow using the CLI
+tags:
+  - typescript sdk
+  - workflow
+  - developer guide
+  - temporal cli
+---
+
+**How to start a Workflow using the CLI**
+
+You can use the Temporal CLI to start a Workflow whether you are using a local development server, Temporal Cloud, or are in a self-hosted environment.
+However, you need to provide additional options to the command when operating with the Temporal Cloud or self-hosted environments.
+
+### Local dev Server
+
+**How to start a Workflow with the Temporal CLI while using the local development server**
+
+Use the Temporal CLI `temporal workflow start` command to start your Workflow.
+
+```shell
+temporal workflow start \
+ --task-queue backgroundcheck-boilerplate-task-queue-local \
+ --type backgroundCheck \
+ --input '"555-55-5555"' \
+ --namespace backgroundcheck_namespace \
+ --workflow-id backgroundcheck_workflow
+```
+
+**Parameters breakdown**
+
+- `--task-queue`: The name of the Task Queue for all the Workflow Execution's Tasks.
+  Unless otherwise specified, Activity Executions use the Workflow Execution's Task Queue name by default.
+- `--type`: This is the Workflow Type name.
+  By default, this is the function name.
+- `--input`: This must be a valid JSON object that can be unmarshaled into the parameter(s) that the Workflow function accepts.
+  Read more about how the Temporal Platform handles your application data in the [Data conversion](/concepts/what-is-a-data-converter) guide.
+- `--namespace`: This is the Namespace that you want to run your Temporal Application in.
+- `--workflow-id`: A [Workflow Id](/concepts/what-is-a-workflow-id) is a custom identifier provided by you.
+  We highly recommend supplying your own Workflow Id with your own naming convention.
+  A [Workflow Id Reuse Policy](/concepts/what-is-a-workflow-id-reuse-policy) enables fine controls over whether Workflow Ids can be reused in the Platform within the Retention Period.
```

Comment:
Since you specify the default for the others, do you want to specify that the default is WORKFLOW_ID_REUSE_POLICY_ALLOW_DUPLICATE which allows the Workflow ID to be reused by another Workflow Execution in the same namespace upon the conclusion of the current one, regardless of whether that resulted in success or failure.